example/duo_admin_policy.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+
+var nopt = require('nopt')
+var duo_api = require('../')
+
+var parsed = nopt({
+  'ikey': [String],
+  'skey': [String],
+  'host': [String]
+}, [], process.argv, 2)
+
+var requirements_met = (parsed.ikey && parsed.skey && parsed.host)
+
+if (!requirements_met) {
+  console.error('Missing required option.\n')
+}
+
+if (parsed.help || !requirements_met) {
+  console.log(function () { /*
+Usage:
+
+    duo_admin_policy.js --ikey IKEY --skey SKEY --host HOST
+
+    Example of making one Policy API call against the Duo service.
+
+Options:
+
+    --ikey    Admin API integration key (required)
+    --skey    Corresponding secret key (required)
+    --host    API hostname (required)
+    --help    Print this help.
+*/ }.toString().split(/\n/).slice(1, -1).join('\n'))
+  if (parsed.help) {
+    process.exit(0)
+  } else {
+    process.exit(1)
+  }
+}
+
+var client = new duo_api.Client(
+  parsed.ikey,
+  parsed.skey,
+  parsed.host,
+  duo_api.SIGNATURE_VERSION_5
+)
+
+let params = {
+  'policy_name': 'api_test_policy',
+  'sections': {
+    'screen_lock': {
+      'require_screen_lock': false
+    }
+  }
+}
+
+let policy_key = ''
+
+client.jsonApiCall(
+  'POST',
+  '/admin/v2/policies',
+  params,
+  function (res) {
+    if (res.stat !== 'OK') {
+      console.error('API call returned error: ' + res.message)
+      process.exit(1)
+    }
+
+    res = res.response
+    policy_key = res.policy_key
+
+    console.log('res = ' + JSON.stringify(res, null, 4))
+
+    // Delete the created policy
+    deletePolicy(policy_key)
+  },
+  5
```

Comment:
would it be better to pass `duo_api.SIGNATURE_VERSION_5` here instead of `5`?

Reply:
Actually, `jsonApiCall` calls `Client.prototype.apiCall` (line 17 in `main.js`) which doesn't take in a signature parameter. The last parameter it takes is the callback. It instead uses the signature passed in to the `Client` constructor. So it seems like no sig version should be passed here if I'm following correctly?

Reply:
Yep, this was extraneous from a previous version. Removed the unneeded parameter and verified everything still works.